app/models/user.rb

```diff
@@ -179,6 +179,15 @@ def get_roles
     result
   end
 
+  ##
+  # Returns registration tickets of the user for the given conference.
+  #
+  # ====Returns
+  # * +Array+ -> registation tickets
+  def registration_tickets conference
```

Comment:
this could be moved inside the [`tickets` association](https://github.com/siddhantbajaj/osem/blob/25bbe207cbd782977f7181738831b4ebfc1ef748/app/models/user.rb#L54). See the _Extensions_ section in the [Rails `has_many` documentation](https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/has_many)